declarative_clusterconfig/managing-app-sets-in-non-control-plane-namespaces.adoc

```diff
@@ -0,0 +1,44 @@
+:_mod-docs-content-type: ASSEMBLY   
+include::_attributes/common-attributes.adoc[]
+[id="managing-app-sets-in-non-control-plane-namespaces"]
+= Managing the application set resources in non-control plane namespaces
+:context: managing-app-sets-in-non-control-plane-namespaces
+
+toc::[]
+
+:FeatureName: `applicationSet` resources in non-control plane namespaces
+include::snippets/technology-preview.adoc[]
+
+By using application sets, you can automate and manage the deployments of multiple Argo CD applications declaratively from a single mono-repository to many clusters at once with greater flexibility.
```

Comment:
```suggestion
By using applicationsets, you can automate and manage the deployments of multiple Argo CD applications declaratively from a single mono-repository to many clusters at once with greater flexibility.
```

Reply:
@varshab1210 and @svghadi As per this guideline - https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting, we have to keep it separate and we are retaining it as is here.